### PR TITLE
Update to d3m v2019.11.10

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Various update to switch to the latest release of Featuretools (0.12.0) and d3m (2019.11.10).

Closes issues #27 and #28

### CODE REVIEW

File: construct_predictions.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ class ConstructPredictionsPrimitive(transformer.TransformerPrimitiveBase[Inputs,
             'id': '8d38b340-f83f-4877-baaa-162f8e551736',
             'version': '0.3.0',
             'name': "Construct pipeline predictions output",
-            'python_path': 'd3m.primitives.data_transformation.construct_predictions.DataFrameCommon',
+            'python_path': 'd3m.primitives.data_transformation.construct_predictions.Common',
```
Comment (kmax12) [inline construct_predictions.py]: do we actually use this file?

Comment (thehomebrewnerd) [reply construct_predictions.py]: I'll have to double check this. The `dockerfile` calls out this being added, and I thought I needed to make this update to resolve an error at one point, but now that I am testing again, I'm not so sure.

Comment (thehomebrewnerd) [reply construct_predictions.py]: This is being added to the container by the `dockerfile`, but that may not be necessary. At least a couple of the tests run without the `ADD` from the `dockerfile`. I can do some more comprehensive testing and delete this file and update the `dockerfile` accordingly if it is no longer needed. Do you know why it was set up this way to begin with?

File: featuretools_ta1/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-__version__ = '0.6.1'
+__version__ = '0.6.2'
```
Comment (kmax12) [inline featuretools_ta1/__init__.py]: can we make this `0.7.0`?

File: setup.py [hunk 3]
```diff
@@ -23,11 +23,11 @@
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=[
         'd3m',
-        'featuretools @ git+https://github.com/featuretools/featuretools.git@2204f0ee302b69e75e096035e5c628f96b14fb78#egg=featuretools',
+        'featuretools',
```
Comment (kmax12) [inline setup.py]: we might need to keep this until we do the release with fixes

File: setup.py [hunk 4]
```diff
@@ -23,11 +23,11 @@
...
     ],
     url='https://gitlab.datadrivendiscovery.org/MIT-FeatureLabs/ta1-primitives',
     entry_points={
         'd3m.primitives': d3m_primitives,
     },
-    version='0.6.1',
+    version='0.6.2',
```
Comment (kmax12) [inline setup.py]: `0.7.0` as mentioned above

File: featuretools_ta1/__init__.py (hunk 2 above)
Comment (thehomebrewnerd) [reply featuretools_ta1/__init__.py]: Updated

File: setup.py (hunk 3 above)
Comment (thehomebrewnerd) [reply setup.py]: Will leave the old version for now and we can revisit depending on where we are by the end of the week.

**Pull request merged**